project/src/main/webapp/script.js

```diff
@@ -220,7 +228,7 @@ function createMapMarker(map, placePosition, placeTitle) {
   })
 }
 
-/** Displays a Google Maps map that allows the user to search fo his location. */
+/** Displays a Google Maps map that allows the user to search for his location. */
```

Comment:
for his location => for their location

Reply:
Done, thanks. Changed in more places as well.